src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -130,6 +133,21 @@ protected virtual void Dispose(bool disposing)
 
         async void InitializeAsync()
         {
+            VirtualFolderPath = string.Empty;
+            try
+            {
+                if (viewModel.Link != null && !string.IsNullOrEmpty(viewModel.PackageName))
+                {
+                    VirtualFolderPath = Path.GetDirectoryName(HttpUtility.UrlDecode(viewModel.Link.AbsolutePath));
+                }
+                else
+                    VirtualFolderPath = FallbackDirectoryName;
+            }
+            catch (Exception ex)
```

Comment:
what exception do you think will be caught here?

Reply:
@mjkkirschner I found a case that `viewModel.Link = "{}"` so it was crashing when trying the next line `viewModel.Link.AbsolutePath` so I added this try catch because I didn't found any other way to validate the `viewModel.Link` content.
With the fix added related to DSVarArgFunction now I expect that viewModel.Link always will contain something valid but just in case I think we should keep this code or if you have any other idea in how to validate  `viewModel.Link = "{}"` please let me know (Link is Uri type, Uri.IsWellFormedUriString returns true).

Reply:
I think it's a good practice to log something including the exception if possible to the dynamo console - if some unanticipated exception starts showing up here atleast we'll know.

Reply:
@mjkkirschner I've added some code for logging the exception in the dynamo console, see the next commit: https://github.com/DynamoDS/Dynamo/pull/13703/commits/5980d151914322d4a1eb6d6965f5e2b116cfc9a4